using-timescaledb/continuous-aggregates.md

```diff
@@ -248,6 +244,111 @@ hypertable with a continuous aggregate.
 
 ---
 
+### Advanced Topics [](advanced-usage)
+
+#### Continuous Aggregates using Integer-Based Time [](create-integer)
+
+Usually, continuous aggregates are defined on a [date/time-type](https://www.postgresql.org/docs/current/datatype-datetime.html) column, but it is
+also possible to create your own custom scheme for handling
+aggregation for tables that are using an integer time
+column. This can be useful if you have tables that use other measures
+of time that can be represented as integer values, such as nanosecond
+epochs, minutes since founding date, or whatever is suitable for your
+application.
+
+As an example, suppose that you have a table with CPU and disk usage
+for some devices where time is measured in
+[microfortnights][fff-system]. Since you are using an integer-valued
+column as time, you need to provide the chunk time interval when
+creating the hypertable. In this case, let each chunk consist of a
+millifortnight (which is about 20 minutes).
+
+```sql
+CREATE TABLE devices(
+  time BIGINT,        -- Timestamp in microfortnights
+  cpu_usage INTEGER,  -- Total CPU usage
+  disk_usage INTEGER, -- Total disk usage
+  PRIMARY KEY (time)
+);
+
+SELECT create_hypertable('devices', 'time',
+                         chunk_time_interval => 1000);
+```
+
+To define a continuous aggregate on a hypertable that is using an integer
+dimension, it is necessary to have a function to get the current
+timestamp in whatever representation that you are using and set it for
+the hypertable using
+[`set_integer_now_func`][api-set-integer-now-func]. The function can
+be defined as a normal PostgreSQL function, but needs to: be
+[`STABLE`][pg-func-stable], take no arguments, and return an integer
+of the same type as the time column in the table. In our case, this
+should suffice:
+
+```sql
+CREATE OR REPLACE FUNCTION current_microfortnight() RETURNS BIGINT
+LANGUAGE SQL STABLE AS $$
+	SELECT CAST(1209600 * EXTRACT(EPOCH FROM CURRENT_TIMESTAMP) / 1000000 AS BIGINT)
+$$;
+
+SELECT set_integer_now_func('devices', 'current_microfortnight');
+```
+
+Once the replacement for current timestamp has been set up, you can
+define the continuous aggregate for the `devices` table.
```

Comment:
I guess it should be `a` continuous aggregate, since it (its shape) wasn't described before.
```suggestion
define a continuous aggregate for the `devices` table.
```